x-pack/plugins/observability_solution/inventory/public/components/grouped_inventory/inventory_panel_badge.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiBadge, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React from 'react';
+
+export function InventoryPanelBadge({
+  name,
+  value,
+  'data-test-subj': dataTestSubj,
+}: {
+  name: string;
+  'data-test-subj'?: string;
+  value: string | number;
+}) {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <div
+      data-test-subj={dataTestSubj}
+      css={css`
+        padding: ${euiTheme.size.base};
+      `}
+    >
+      <span
+        css={css`
+          font-weight: ${euiTheme.font.weight.bold};
+          margin-right: ${euiTheme.size.s};
+        `}
+      >
+        {name}:
+      </span>
+      <EuiBadge color="hollow">{value}</EuiBadge>
+    </div>
```

Comment:
Can you use <EuiFlexGroup>... here instead of manually adding all these spaces between elements?

You can also use <EuiText>{name}</EuiText>

Reply:
It's not good practice to rely on whitespace for spacing between elements, given the separation is between visual elements not text.

Reply:
nvm, fixed this because I didn't see one of your edits.